documentation/configuration.rst

```diff
@@ -671,4 +671,13 @@ FLEXMEASURES_API_SUNSET_LINK
 
 Allow to override the default sunset link for your clients.
 
-Default: ``None`` (defaults are set internally for each sunset API version, e.g. ``"https://flexmeasures.readthedocs.io/en/v0.13.0/api/v2_0.html"`` for v2.0)
\ No newline at end of file
+Default: ``None`` (defaults are set internally for each sunset API version, e.g. ``"https://flexmeasures.readthedocs.io/en/v0.13.0/api/v2_0.html"`` for v2.0)
+
+Perforamance optimizations
+----------------------------
+
+FLEXMEASURES_MVIEW_UPDATE_INTERVAL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Interval in minutes to refresh the materialized views in the background.
```

Comment:
This sentence should help a little more as to what is meant. I suggest:  "... to refresh the materialized db view which caches the most recent beliefs at a given point in time (for faster queries).